tools/wptrunner/wptrunner/wptrunner.py

```diff
@@ -245,6 +245,9 @@ def run_tests(config, test_paths, product, **kwargs):
                             if test.testdriver and not executor_cls.supports_testdriver:
                                 logger.test_start(test.id)
                                 logger.test_end(test.id, status="SKIP")
+                            elif test.jsshell and not executor_cls.supports_jsshell:
+                                logger.test_start(test.id)
```

Comment:
Does the jsshellness change the test id here? If not we are going to get multiple results for the "same" test.